### DISCUSSION TITLE
gothic-win64-v1.0.1177

### DISCUSSION THREAD

**Discussion by Try** [Try/OpenGothic#183] [General]
![изображение](https://user-images.githubusercontent.com/122894/127773629-b97a2a3f-5f94-4d61-9ead-a3c1c1ad006d.png)

* General
  * Added bounce behaviour for npc collision
  * Improve swimming and diving
  * Recognize spellType 
  * Improve torch mechanics
  * Overall CPU usage optimization

* Script-api 
  * Implement AI_Flee
  * Implement AI_LookAt
  * Implement AI_PointAt, AI_PointAtNpc, AI_StopPointAt
  * Implement Npc_IsInState
  * Implement Wld_GetFormerPlayerPortalGuild
  * Implement MOBSI::conditionFunc
  * Implement Npc_GetDistToPlayer

* Bugfix
  * Fixed wrong output of Npc_GetEquippedMeleeWeapon
  * Fixed cliff sliding 
  * Fixed wall tonneling
  * Fixed ground alignment for monsters
  * Fixed animation solver cache

<hr /><em>This discussion was created from the release <a href='https://github.com/Try/OpenGothic/releases/tag/v0.48'>gothic-win64-v1.0.1177</a>.</em>

**Comment by clort81** [comment]
Excited to try, but panfrost-vulkan driver progress is slow.

**Comment by czarnyafgan** [comment]
Any ETA for next version release ?

Can't wait!
